doc/tutorials/rf.rst

```diff
@@ -1,9 +1,9 @@
 #########################
-Random Forests in XGBoost
+Random Forests(TM) in XGBoost
```

Comment:
Minitab owns a trademark for the term Random Forests ([source](https://en.wikipedia.org/wiki/Random_forest)). It's standard to write the trademark symbol once at the beginning of each document that mentions a trademarked term. When I read about [RST ](https://docutils.sourceforge.io/docs/ref/rst/directives.html), (TM) seems to convert to ™.

BTW, there's another XGBoost document that uses the ™ symbol with Random Forests, but it would be better to change it later so that it only uses the ™ symbol the first time.

Reply:
Thanks for the explanation.  Good to learn something new ;-)